I am getting joda time errors when trying to run this.

error while loading DateTime, Missing dependency 'class org.joda.convert.FromString', required by /Users/kiivihal/Experiments/gitHub/delving/culture-hub/modules/press-1.0.24.P/lib/joda-time-2.0.jar(org/joda/time/DateTime.class)
error while loading AbstractInstant, Missing dependency 'class org.joda.convert.ToString', required by /Users/kiivihal/Experiments/gitHub/delving/culture-hub/modules/press-1.0.24.P/lib/joda-time-2.0.jar(org/joda/time/base/AbstractInstant.class)
two errors found
error while loading DateTime, Missing dependency 'class org.joda.convert.FromString', required by /Users/kiivihal/Experiments/gitHub/delving/culture-hub/modules/press-1.0.24.P/lib/joda-time-2.0.jar(org/joda/time/DateTime.class)
error while loading AbstractInstant, Missing dependency 'class org.joda.convert.ToString', required by /Users/kiivihal/Experiments/gitHub/delving/culture-hub/modules/press-1.0.24.P/lib/joda-time-2.0.jar(org/joda/time/base/AbstractInstant.class)
two errors found
2011-11-19 10:10:53 play [ERROR] 

@68eckol8b
Internal Server Error (500) for request GET /

Compilation error
The file  could not be compiled. Error raised is : error while loading DateTime, Missing dependency 'class org.joda.convert.FromString', required by /Users/kiivihal/Experiments/gitHub/delving/culture-hub/modules/press-1.0.24.P/lib/joda-time-2.0.jar(org/joda/time/DateTime.class)

play.exceptions.CompilationException: error while loading DateTime, Missing dependency 'class org.joda.convert.FromString', required by /Users/kiivihal/Experiments/gitHub/delving/culture-hub/modules/press-1.0.24.P/lib/joda-time-2.0.jar(org/joda/time/DateTime.class)
    at play.scalasupport.ScalaPlugin.compilationException(ScalaPlugin.scala:138)
    at play.scalasupport.ScalaPlugin.detectClassesChange(ScalaPlugin.scala:121)
    at play.plugins.PluginCollection.detectClassesChange(PluginCollection.java:413)
    at play.Play.detectChanges(Play.java:612)
    at play.Invoker$Invocation.init(Invoke

--- a/conf/dependencies.yml
+++ b/conf/dependencies.yml
@@ -7,6 +7,7 @@ require:
     - delving -> scala 0.9.1_20111028
     - delving -> press 1.0.24.P
     - play -> sass 1.1
+    - org.joda -> joda-convert 1.1
     - commons-httpclient -> commons-httpclient 3.1
     - net.liftweb -> lift-json-ext_2.8.1 2.4-M4:
         exclude:

